internal/query/projection/org/owner/view.go

```diff
@@ -0,0 +1,425 @@
+package owner
```

Comment:
maybe we have a better name than view.go

Reply:
renamed to projection